### PR TITLE
Program arguments

### PR DESCRIPTION
**Pull request by DavidGeismarLtd**
dynamic arguments for programs

### CODE REVIEW

File: app/models/argument.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Argument < ApplicationRecord
+  extend Enumerize
+  ICONS = %i[clock chat check bookmark].freeze
+  belongs_to :program
+
+  validates :description, :icon, presence: true
+
+  enumerize :icon, in: ICONS
```
Comment (ManyBaby) [inline app/models/argument.rb:10]:
```suggestion
  enum icon: ICONS
```

[the remark this answers is not in the document]
Comment (DavidGeismarLtd) [reply app/models/argument.rb:17]: ok Ill remove the gem

[the remark this answers is not in the document]
Comment (DavidGeismarLtd) [reply app/models/argument.rb]: What is exactly the use case for enumerize vs enum @ManyBaby  ?

[the remark this answers is not in the document]
Comment (ManyBaby) [reply app/models/argument.rb]: The old enum was very basic but it integrated most enumerize feature with rails getting more powerful.

**Pull request merged**